src/Microsoft.DotNet.Build.Tasks/Targets/coreAssembly.rsp

```diff
@@ -0,0 +1,2 @@
+﻿# CSC requires the runtimemetadataversion flag when building a core assembly (containing System.Object)
+/runtimemetadataversion:v4.0.30319
```

Comment:
I see how it's done: https://github.com/dotnet/coreclr/blob/4cf8a6b082d9bb1789facd996d8265d3908757b2/src/mscorlib/GenerateCompilerResponseFile.targets